### ISSUE TITLE
Catalina cmd + V and cmd + Shift + V not working

### ISSUE THREAD

**Issue by LisaNewcomb** [TermiT/Flycut#185]
I have enabled Flycut in System Pref, privacy & accessibility, restarted the computer, chrome and redownloaded flycut but I am unable to use it as I have been.

Neither function works, though when I review the clips in the flycut menu the cmd + c copies are there, they aren't pasting into G docs. 

Macbook pro, catalina

**Comment by TermiT** [comment]
Try to remove and re-add Flycut to accessibility

**Comment by TermiT** [comment]
Only doesn't work with G docs?

**Comment by LisaNewcomb** [comment]
No, it's everywhere (tried just now in this text box). It just pastes whatever the last thing was I copied, even though if I click the dropdown from the icon in top bar on my mac, it shows it's recording other cmd+c commands. 

I'll try I guess uninstalling and reinstalling and then undoing and adding to accessibility? I've already added to accessibility so that's not the issue I don't think. Thanks!

**Comment by TermiT** [comment]
I've got a few dozen emails with that issue. It caused by a MacOSX bug. Re-adding to accessibility help for all of them. You don't need to re-uninstall just remove (not unchecking — removing)  it from accessibility and add again. 

Also, you may want to disable "Sticky Bezel" in preferences.

**Comment by LisaNewcomb** [comment]
Ok, I'll try that. Thanks.

**Comment by nartcan** [comment]
> I've got a few dozen emails with that issue. It caused by a MacOSX bug. Re-adding to accessibility resolved the issue for all of them. You don't need to re-uninstall just remove (not unchecking — removing) it from accessibility and add again.
> 
> Also, you may want to disable "Sticky Bezel" in preferences.

Hi, I've faced the same problem with Catalina and El Capitan.

It solved the issue for Catalina. But for El Capitan, it didn't work.

**Comment by jvsteiner** [comment]
>Try to remove and re-add Flycut to accessibility
This worked for me
